### PR TITLE
Wqp 1559 - Update iow-ecs-pipeline to the latest

### PR DESCRIPTION
**Pull request by mbucknell**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: pipeline.yml [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
-# set to a low priority; services are given higher priority.
-rulePriority: 503
 # port that is exposed by your container
 containerPort: 5050
 # string parameter specifying a path that ALB should use to verify that your application is alive
 healthCheck: "/robots.txt"
 # string parameter that tells the ALB how to route requests to the containers (e.g. /my-app/*); this forms a listener rule for the load balancer
 contextPath: "/*"
-# the Jenkins credential ID to be use when pushing commits
-gitRepoCredentialsId: "Jenkins-GitHub-Read-Write-Token"
```
Comment (ayan-usgs) [inline pipeline.yml:27]: This particular credential is still useful. It's used to push incremented version numbers/tags back into the GitHub repo.

File: pipeline.yml [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
 applicationName: "wqp_ui"
-# integer representing the order of precedence of the ALB rule.
-# set to a low priority; services are given higher priority.
-rulePriority: 503
 # port that is exposed by your container
 containerPort: 5050
 # string parameter specifying a path that ALB should use to verify that your application is alive
 healthCheck: "/robots.txt"
 # string parameter that tells the ALB how to route requests to the containers (e.g. /my-app/*); this forms a listener rule for the load balancer
 contextPath: "/*"
```
Comment (ayan-usgs) [inline pipeline.yml:24]: This is superseded by line 31.

File: pipeline.yml (hunk 1 above)
Comment (mbucknell) [reply pipeline.yml:27]: I moved this to line 8. A little rearrangement in an attempt to make sure I got all the required parameters.

Comment (ayan-usgs) [reply pipeline.yml:27]: Ah. Brilliant!

**Pull request merged**